index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Selcuk Guler's Personal Portfolio website</title>
+  </head>
+  <body>
+    <main>
+      <div id="topbar">
+        <div id="appbar">
+          <div id="timer-cont">
+            <div id="timer-text">9:41</div>
+          </div>
+          <div id="statusbar">
+          </div>
+        </div>
+        <div class="container" id="toolbar">
+          <div id="logo-placeholder"></div>
+          <nav id="hamburger-cont">
+            <div id="hamburger-icon"></div>
+          </nav>
+        </div>
```

Comment:
- [ ] This is the appropriate place for the header tag. The haeder tag houses the logo and the hamburger of your page on the mobile. The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation(in this case, your hamburger). Kindly change the generic `div` with the `header` tag

###  Items inside your header
![your-toolbar](https://github.com/s2jack/create-portfolio-website1/assets/26597432/95790ec4-7761-4659-b53b-07e2435bd4e2)

### Items to be inside the header tag
![project-rqt](https://github.com/s2jack/create-portfolio-website1/assets/26597432/4019be80-30bb-4388-8716-3f07c1995104)

- [ ] As mentioned by the previous reviewer, it is not a good practice for the  `header` tag to be inside the main tag. Kindly address that too.